src/Symfony/Component/Validator/Constraints/UniqueValidator.php

```diff
@@ -51,4 +55,27 @@ public function validate($value, Constraint $constraint)
             $collectionElements[] = $element;
         }
     }
+
+    private function getValueNormalizer(Unique $unique)
+    {
+        $normalizer = $unique->valueNormalizer;
+
+        if (null === $normalizer) {
+            return static function ($value) {
+                return $value;
+            };
+        }
+
+        if (\is_callable($normalizer)) {
+            return $normalizer;
+        }
+
+        if (\is_array($normalizer) && !\is_callable($normalizer) && isset($normalizer[0]) && \is_object($normalizer[0])) {
+            $normalizer[0] = \get_class($normalizer[0]);
+
+            return $normalizer;
+        }
```

Comment:
I don't understand what you are doing here. 🤔

Reply:
Yeah, looks like I wanted to check all the cases and overengineered a bit here. Simplified :) thanks